getting-started/installation.md

```diff
@@ -22,6 +22,10 @@ Therefore the only officially supported way of installation is by using a [manua
 The installation step for `npm install --only=prod --omit=dev` will take a very long time, often with little or no terminal response! For the RPi3 this is **~10** minutes and for the Rpi2 **~25** minutes. Do not interrupt or you risk getting a :broken_heart: by Raspberry Jam.
 :::
 
+::: warning NOTE
+You may receive a High Security Alert for npm during the installation of MagicMirror verion 2.19.0. Please ignore the alert as there othing we can do about it. The audit fix causes more trouble than it fixes and npm 8 has not been tested with this version of MagicMirror.
```

Comment:
I would omit `verion 2.19.0` because this is not version specific (otherwise fix typo `verion` vs. `version`). Another typo `othing` vs. `nothing`.